crates/ruma-events-macros/src/lib.rs

```diff
@@ -36,10 +36,12 @@ mod event_parse;
 /// # Examples
 ///
 /// ```ignore
+/// # // HACK: "ignore" the doctest here because it needs ruma_events, which cannot be imported
```

Comment:
Actually, this can be made to work. Cyclic dependencies are allowed as long as at least one of them is a `dev-dependency`, though that is kind of annoying when publishing. There's another possible fix though: Move all of these docs to the macro re-exports in `ruma-events`.

Reply:
I'll do the latter one 👍